### ISSUE TITLE
Are these supposed to be the right direction icons?

### ISSUE THREAD

**Issue by Hacksaurus** [numixproject/numix-icon-theme#1315]
I am using numix-circle-icon-theme-git from Arch Linux and actually a while ago when I updated either Numix or Thunar, the icons up top changed. So I just wanted to verify if these are the correct icons.

![screenshot_2018-09-27_05-31-38](https://user-images.githubusercontent.com/24832107/46146177-fe850680-c216-11e8-8a79-65e58e97978b.png)

P.S. The program xfce4-clipman-plugin also uses this arrow icon, so if this is an issue also check with that too!

**Comment by palob** [comment]
Can you please show a screenshot with another theme applied?

**Comment by Hacksaurus** [comment]
If you mean GTK Theme, this is the Raliegh Theme
![screenshot_2018-09-27_15-47-17](https://user-images.githubusercontent.com/24832107/46178675-a4af2b80-c26c-11e8-8d9c-3341f080e721.png)

**Comment by palob** [comment]
No, I mean the icon theme.

**Comment by Hacksaurus** [comment]
This is with the GNOME Icon Theme, I think you are checking if the direction button icons change and they do.
![screenshot_2018-09-28_15-21-02](https://user-images.githubusercontent.com/24832107/46236119-3f257280-c332-11e8-9436-f0d34d6747ee.png)

**Comment by palob** [comment]
Sorry, I still don't understand what's the issue. From what I see the GNOME icons point to the same directions.
Or are you wondering about the different shape (opening angle in particular). That's by design.

**Comment by Hacksaurus** [comment]
Really? The House in particular looks very lopsided and blury, compared to the other icon siet

**Comment by palob** [comment]
I took a look and all our ``go-home`` icons are aligned to the pixel grid so they shouldn't be blurry as long as they aren't scaled to the wrong size.

Anyway from what I see you are not using upstream Numix. Those folder icons are not ours.

**Comment by Hacksaurus** [comment]
I am actually using the Numix-circle-icon, the git page for that actually told me to post issues on this git. Sorry for not making that clear.

**Comment by Hacksaurus** [comment]
![image](https://user-images.githubusercontent.com/24832107/46484118-f6e0d700-c7ad-11e8-8707-9904a59e538b.png)
This picture is the direction icons I expected FYI I can't seem to find the icons that are being presented in Thunar in any of the Numix icon packs. This may be a problem with Thunar not using the correct icon name? Or using their own icons, I'm not 100% sure.

**Comment by Foggalong** [comment]
Yeah those in the latest screenshot are the icons I get with Thunar too. Which Thunar version is having the issue?

**Comment by Hacksaurus** [comment]
@Foggalong I'm on version 1.8.2

**Comment by palob** [comment]
What did you cahnge to get the circlish icons in the last screenshot?

**Comment by Foggalong** [comment]
@palob Time travel ;) (look at the conky date)

**Comment by Hacksaurus** [comment]
Also that wasn't my screenshot I literally just googled thunar with numix icon theme XD
But I finally found a screenshot of my own. This is a little more dated... 2017/9/8
![screenshot_2017-09-08_17-30-01](https://user-images.githubusercontent.com/24832107/46489580-a2902400-c7ba-11e8-9616-b1f9aaa31710.png)

**Comment by palob** [comment]
I did another search and it turns out we've got those circular designs for the ``go-*`` icon for sizes 32px and above. This doesn't make too much sense. The sub-32px-design is a symbolic icon actually.

For most other action icons we use the same design across all sizes (only changing details shown). 

Maybe this was an exception so the icons do not stand out in GNOME where interface icons are symbolic-only these days?

**Comment by Hacksaurus** [comment]
![screenshot_2018-10-04_16-54-49](https://user-images.githubusercontent.com/24832107/46509208-39c69d00-c7f6-11e8-9ccb-a5d461a76a50.png)
I did a search on my side, and it appears that icons are what they are in the base Numix Icons (Which Numix-Circle-Icons should be symbolically linking them I believe) BTW I am on Arch Linux and I have installed the `numix-circle-icon-theme-git` and the `numix-folders-git` This may be also an issue on the AUR pacakge, but at the same time it is a git package so it should be automatically updated if I remember correctly.

**Comment by palob** [comment]
As I said, it depends on the icon size you look at. Btw Numix Circle is dependent on the base theme as it only provides app icons.

**Comment by Hacksaurus** [comment]
@palob I understand that, so Numix-Circle-Icons should be symbolically linking the all other types of icons to the Numix Base icons. 
![screenshot_2018-10-05_08-19-49](https://user-images.githubusercontent.com/24832107/46544135-9f0ea280-c877-11e8-8326-680147d15dfd.png)
![screenshot_2018-10-05_08-20-05](https://user-images.githubusercontent.com/24832107/46544134-9f0ea280-c877-11e8-839d-adbaf1b3c6cf.png)
From what I seeing here, Numix-Circle is only linking the panel icons. And this is true with every sized icon up until 48. Where there is only the apps folder icons as u mentioned.
![screenshot_2018-10-05_08-37-17](https://user-images.githubusercontent.com/24832107/46544970-e433d400-c879-11e8-8353-651ae46c8d88.png)
I feel like this might be the cause of the problem

**Comment by palob** [comment]
I didn't check yet if we (or Thunr) changed something in the time since the old screenshots but technically there is no issue as the icons show up as expected/intended.

**Comment by palob** [comment]
The AUR package is probably outdated which would mean we changed something (can't check atm).
The icons are named **go-...** and can be found in **Numix/<size>/actions** where **<size> = {16,22,24}**.

**Comment by Hacksaurus** [comment]
@palob @Foggalong I talked to the Thunar guys and it seems that it was a change from their side! They have changed icons from regular to symbolic. https://forum.xfce.org/viewtopic.php?pid=49866#p49866
So I had to change my gtk settings to this:
```/* thunar toolbar icons */
.thunar {
	-gtk-icon-style: regular;
	-gtk-icon-transform: scale(1.3);
}```
So now my question is that, why are the symbolic icons so ugly?

**Comment by palob** [comment]
Does ``scale(1.3)`` mean the icons are simply scaled by a factor of *1.3*?
If so, this is the culprit. Scaling by non-integer values will result in blurry icons.

**Comment by Hacksaurus** [comment]
@palob Sorry for not updating that, this is my current settings: 
```
.thunar > grid > toolbar * {
	-gtk-icon-style: regular;
	-gtk-icon-transform: scale(1.0);
}
```

From what I discovered the scaling of the icons is actually not 1, this is the screenshot of what it looks like now:
![screenshot_2018-10-08_14-00-02](https://user-images.githubusercontent.com/24832107/46633430-76e6a400-cb02-11e8-868d-1db51af410f2.png)
This is with the symbolic icons (still with a scaling with 1):
![screenshot_2018-10-08_14-00-44](https://user-images.githubusercontent.com/24832107/46633491-97aef980-cb02-11e8-8715-2c5baa029fc1.png)

So Thunar messed up with their scaling, but I now I know that there was a shift from regular icons to symbolic icons I think its time to make another issue protesting about why the symbolic icons look weird!

**Comment by palob** [comment]
I don't know what you mean by weird but the non-symbolic circlish icons are sized 22/24px. The symbolic icons on the other hand are designed for a size of 16px and are not to be used as replacement for 24px icons because scaling by 1.5 will make those bluury.

**Comment by Hacksaurus** [comment]
@palob What I mean by "weird" is just the design in general. For instance why is the home button all in a rectangle? And the angles of arrows are kinda of horrible :-1:

**Comment by palob** [comment]
That's not a nice way to approach us for a redesign. Feel free to open a new issue to put the design up to discussion.
